editor/src/components/canvas/canvas-strategies/strategies/reparent-utils.ts

```diff
@@ -538,6 +538,36 @@ function pasteNextToSameSizedElement(
   return null
 }
 
+function pasteIntoNextGridCell(
+  copyData: ParsedCopyData,
+  selectedViews: NonEmptyArray<ElementPath>,
+  metadata: ElementInstanceMetadataMap,
+): ReparentTargetForPaste | null {
+  if (selectedViews.length === 0) {
+    return null
+  }
+  const selectedView = lastOfNonEmptyArray(selectedViews)
+
+  // if the copied elements are grid cells and the target is a grid cell, paste next to it
+
+  const copyDataElementsAreGridCells = copyData.elementPaste.every(({ originalElementPath }) =>
+    MetadataUtils.isGridCell(copyData.originalContextMetadata, originalElementPath),
+  )
+  if (!copyDataElementsAreGridCells) {
+    return null
+  }
+
+  const targetIsGridCell = MetadataUtils.isGridCell(metadata, selectedView)
```

Comment:
just a small unrelated naming nitpick - the element itself should probably be called "grid child" (since a "grid cell" is usually the actual cell and not the element being copied in this case)

Reply:
true, but we should also update the method name (`isGridChild`). We have a bunch of naming cleanup to do, (e.g. `tracks` vs `dimensions`)

Reply:
[`c170a70` (#6516)](https://github.com/concrete-utopia/utopia/pull/6516/commits/c170a70c7ab22c9b3a1434a01fbf56cfd7cfe786)